<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: transparent;

border-image: url(:pics/Alisochka)
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">#stackedWidget{
background:transparent;
}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Intro">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>448</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>198</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QTextBrowser" name="Greeting">
              <property name="styleSheet">
               <string notr="true">#Greeting{
background-color: transparent;
border-image: url(:pics/dialog.png);
background: none;
border: none;
background-repeat: none;
padding:10px;
}</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="ConfirmIntro">
              <property name="styleSheet">
               <string notr="true">#ConfirmIntro{
background-color: transparent;
border-image: url(:pics/dialog.png);
background: none;
border: none;
background-repeat: none;
padding:10px;
}
#ConfirmIntro:hover{
border-width:3px;
}</string>
              </property>
              <property name="text">
               <string>Так</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="interface_2">
         <property name="styleSheet">
          <string notr="true">#intererface_2{
backgrond:transparent;
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTabWidget" name="tabWidget">
              <property name="styleSheet">
               <string notr="true"> #tabWidget{
opacity:1;
}</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="Sort">
               <attribute name="title">
                <string>А давайте посортуємо</string>
               </attribute>
               <widget class="QListView" name="CurrentContainer">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>40</y>
                  <width>261</width>
                  <height>411</height>
                 </rect>
                </property>
               </widget>
               <widget class="QListView" name="SortedContainer">
                <property name="geometry">
                 <rect>
                  <x>730</x>
                  <y>30</y>
                  <width>261</width>
                  <height>411</height>
                 </rect>
                </property>
               </widget>
               <widget class="QPushButton" name="GenArr">
                <property name="geometry">
                 <rect>
                  <x>440</x>
                  <y>120</y>
                  <width>131</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Generate</string>
                </property>
               </widget>
               <widget class="QSpinBox" name="ContainerSize">
                <property name="geometry">
                 <rect>
                  <x>440</x>
                  <y>40</y>
                  <width>131</width>
                  <height>27</height>
                 </rect>
                </property>
               </widget>
               <widget class="QComboBox" name="typeData">
                <property name="geometry">
                 <rect>
                  <x>440</x>
                  <y>80</y>
                  <width>131</width>
                  <height>26</height>
                 </rect>
                </property>
                <item>
                 <property name="text">
                  <string>int</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::string</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::vector&lt;int&gt;</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Ministry</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::vector&lt;Ministry&gt;</string>
                 </property>
                </item>
               </widget>
               <widget class="QComboBox" name="typeSort">
                <property name="geometry">
                 <rect>
                  <x>440</x>
                  <y>340</y>
                  <width>131</width>
                  <height>26</height>
                 </rect>
                </property>
                <item>
                 <property name="text">
                  <string>Select </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Quick</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Merge</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bucket</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Insert</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Heap</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Radix</string>
                 </property>
                </item>
               </widget>
               <widget class="QPushButton" name="SortArr">
                <property name="geometry">
                 <rect>
                  <x>440</x>
                  <y>380</y>
                  <width>131</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Sort</string>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="Container">
               <attribute name="title">
                <string>Заповнимо ж наші контейнери</string>
               </attribute>
               <widget class="QListView" name="currentContainer">
                <property name="geometry">
                 <rect>
                  <x>750</x>
                  <y>10</y>
                  <width>301</width>
                  <height>461</height>
                 </rect>
                </property>
               </widget>
               <widget class="QLineEdit" name="Elapsed">
                <property name="geometry">
                 <rect>
                  <x>750</x>
                  <y>480</y>
                  <width>301</width>
                  <height>26</height>
                 </rect>
                </property>
               </widget>
               <widget class="QPushButton" name="NewSession">
                <property name="geometry">
                 <rect>
                  <x>490</x>
                  <y>30</y>
                  <width>181</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Новий контейнер</string>
                </property>
               </widget>
               <widget class="QComboBox" name="comboBox">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>30</y>
                  <width>161</width>
                  <height>26</height>
                 </rect>
                </property>
                <item>
                 <property name="text">
                  <string>int</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::string</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::vector&lt;int&gt;</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Ministry</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>std::vector&lt;Ministry&gt;</string>
                 </property>
                </item>
               </widget>
               <widget class="QComboBox" name="comboBox_2">
                <property name="geometry">
                 <rect>
                  <x>230</x>
                  <y>30</y>
                  <width>231</width>
                  <height>26</height>
                 </rect>
                </property>
                <item>
                 <property name="text">
                  <string>Double linked list</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>AVL tree</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Splay tree</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Red black tree</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Coalesced hash</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Set</string>
                 </property>
                </item>
               </widget>
               <widget class="QPushButton" name="GenerateValue">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>230</y>
                  <width>291</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Згенеруванувати нове значення</string>
                </property>
               </widget>
               <widget class="QTextBrowser" name="Setup">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>90</y>
                  <width>621</width>
                  <height>91</height>
                 </rect>
                </property>
               </widget>
               <widget class="QPushButton" name="InsertContainer">
                <property name="geometry">
                 <rect>
                  <x>360</x>
                  <y>230</y>
                  <width>311</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Вставити в контейнер</string>
                </property>
               </widget>
               <widget class="QTextEdit" name="textEdit">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>280</y>
                  <width>621</width>
                  <height>121</height>
                 </rect>
                </property>
               </widget>
               <widget class="QPushButton" name="Erase">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>430</y>
                  <width>621</width>
                  <height>26</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Видалити з контейнера</string>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
